### PR TITLE
PIMS-54

### PR DESCRIPTION
**Pull request by jdtoombs**
Transitions already enabled.

_**Transferred within GRE**_ already part of SPL grouping (#896), added **_On Hold_** to part of SPL grouping.

### CODE REVIEW

**Comment by Fosol** [comment]
We definitely want them to be grouped in SPL.

**Comment by jdtoombs** [comment]
> We definitely want them to be grouped in SPL.

@Fosol 

Done, I did notice the _**PimsContextModelSnapshot.cs**_ had some weird behaviour after generating the migration - does this look correct to you?

File: backend/dal/Migrations/v01.14.01/Down/PostDown/02-WorkflowProjectStatus.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PRINT 'Delete WorkflowProjectStatus - SPL --> On Hold'
+
+DELETE FROM dbo.[WorkflowProjectStatus]
```
Comment (Fosol) [inline backend/dal/Migrations/v01.14.01/Down/PostDown/02-WorkflowProjectStatus.sql:3]: We can't remove this unless there are no projects currently using it.  Additionally, why are you deleting it?  It looks like your script below just adds it back?  Is PIMS-54 enabling the ability for "Not in SPL" to be put "On Hold", or "Transferred within the GRE"?

Comment (jdtoombs) [reply backend/dal/Migrations/v01.14.01/Down/PostDown/02-WorkflowProjectStatus.sql:3]: I did this with a rollback in mind (I am probably misunderstanding the down scripts). But yes PIMS-54 is to enable Not in SPL to be put On Hold or Transferred within the GRE. The transitions are already enabled and Transferred within the GRE is part of SPL already, I just had to add On Hold to the grouping

Comment (Fosol) [reply backend/dal/Migrations/v01.14.01/Down/PostDown/02-WorkflowProjectStatus.sql:3]: I think I was looking at the scripts backwards.  Should have noticed this was a down script.

**Review by Fosol — APPROVED** [review]

**Pull request merged**